src/lang/fi.json

```diff
@@ -173,4 +173,4 @@
         "EXPORT_USER_DATA": "Vie tiedot",
         "EXPORT_USER_DATA_DESCRIPTION": "Voit ladata kaikki sinusta kerätyt tiedot."
     }
-}
\ No newline at end of file
+}
```

Comment:
imo *ei-admin funktionaalisuus* is not very good finnish (not too fond of the english equivalent either). not sure if the header is needed anyway, i would either not let admin users see anything else than admin stuff, or just have the button there but maybe using a different colour or sth.